axelor-base/src/main/java/com/axelor/apps/base/service/DMSServiceImpl.java

```diff
@@ -90,6 +108,48 @@ public void addLinkedDMSFiles(List<? extends Model> entityList, Model entityMerg
     }
   }
 
+  @Override
+  public void unzip(String zipFilePath, Model model) throws AxelorException {
+    try {
+      Map<String, DMSFile> dmsParentsMap = new HashMap<>();
+      DMSFile dmsRoot = getDMSRoot(model);
+      DMSFile dmsHome = getDMSHome(model, dmsRoot);
+      dmsParentsMap.put(File.separator, dmsHome);
+
+      ZipInputStream zis = new ZipInputStream(new FileInputStream(zipFilePath));
```

Comment:
please use try-with-resource statement, currently if we get an exception, the stream will never be closed.